Dockerfile

```diff
@@ -0,0 +1,43 @@
+# syntax = docker/dockerfile:1.2
+FROM ubuntu:20.04 as wheels
+ENV DEBIAN_FRONTEND=noninteractive
+
+LABEL Description="Dallinger base docker image" Version="1.0"
+
+EXPOSE 5000
+
+# Install build dependencies
+RUN apt-get update && \
+    apt-get install -y libpq-dev python3-pip python3-dev enchant tzdata pandoc && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY . /dallinger
+WORKDIR /dallinger
+
+RUN --mount=type=cache,target=/root/.cache/pip \
+    mkdir /wheelhouse && \
+    python3 -m pip wheel --wheel-dir=/wheelhouse -r requirements.txt .
+
+FROM ubuntu:20.04
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install runtime dependencies
+RUN apt-get update && \
+    apt-get install -y libpq5 python3-pip enchant tzdata --no-install-recommends && \
```

Comment:
I just added some chromedriver stuff to ci.yml in order to run bots now that PhantomJS is deprecated. I think it would probably makes sense to also include Chrome and Chromedriver in the docker build.

Reply:
Including chrome and chromedriver would increase the size of the base docker image considerably. There are many reasons the image is best kept as lean as possible.

I need to look into bots to understand in which scenarios chrome and chromedriver are needed. But it's my understanding most of the time they're not.

If they're only needed during development we could maintain two docker images, `dallinger/dallinger` and `dallinger/dallinger-dev`.

Reply:
Adding chrome to the current image would increase image size by 412 Mb uncompressed. If you run this command to try and install it you'll see the long list of dependencies needed:
```
docker run --rm dallinger/dallinger bash -c '
apt update
apt install -y wget
wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
SIZE_BEFORE=$(du -scx /)
echo Size before installing chrome: $SIZE_BEFORE
apt install -y --no-install-recommends ./google-chrome-stable_current_amd64.deb
echo Size before installing chrome: $SIZE_BEFORE
echo Size after installing chrome:
du -scx /
'
```
and the increase in the image size:
```
Size before installing chrome: 561840 / 561840 total
Size after installing chrome:
973840	/
973840	total
```

Reply:
Generally speaking if an experiment uses bots (not many do, I think, but who knows) then the bots are run by a worker instance IIRC. Even when experiments do use bots, they don't necessarily require chromedriver.  There's a "high performance" bot base class that people can use which just makes API calls directly instead of going through the browser. So maybe it could be an option that's disabled by default? Probably nothing to worry about for now given the image size implications.